source/core/ScriptHookVDotNet.vcxproj

```diff
@@ -154,31 +155,8 @@
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <!-- Compile C# source files -->
-  <Target
-    Name="CsCompile"
-    BeforeTargets="ClCompile"
-    Inputs="@(CsCompile)"
-    Outputs="$(IntDir)$(TargetName).netmodule">
-    <Csc
-      Sources="@(CsCompile)"
-      AllowUnsafeBlocks="%(CsCompile.AllowUnsafeBlocks)"
-      DebugType="%(CsCompile.DebugType)"
-      DefineConstants="%(CsCompile.DefineConstants)"
-      DisabledWarnings="%(CsCompile.NoWarn)"
-      DocumentationFile="$(IntDir)$(TargetName).xdc"
-      EmitDebugInformation="%(CsCompile.DebugSymbols)"
-      LangVersion="%(CsCompile.LangVersion)"
-      Optimize="%(CsCompile.Optimize)"
-      OutputAssembly="$(IntDir)$(TargetName).netmodule"
-      PdbFile="$(IntDir)$(TargetName).netmodule.pdb"
-      TargetType="Module"
-      TreatWarningsAsErrors="%(CsCompile.TreatWarningsAsErrors)"
-      Utf8Output="%(CsCompile.Utf8Output)"
-      WarningLevel="%(CsCompile.WarningLevel)"
-      WarningsAsErrors="%(CsCompile.WarningsAsErrors)"
-      WarningsNotAsErrors="%(CsCompile.WarningsNotAsErrors)"
-      ToolExe="$(CscToolExe)"
-      ToolPath="$(CscToolPath)" />
+  <Target Name="CsCompile" BeforeTargets="ClCompile" Inputs="@(CsCompile)" Outputs="$(IntDir)$(TargetName).netmodule">
+    <Csc Sources="@(CsCompile)" AllowUnsafeBlocks="%(CsCompile.AllowUnsafeBlocks)" DebugType="%(CsCompile.DebugType)" DefineConstants="%(CsCompile.DefineConstants)" DisabledWarnings="%(CsCompile.NoWarn)" DocumentationFile="$(IntDir)$(TargetName).xdc" EmitDebugInformation="%(CsCompile.DebugSymbols)" LangVersion="%(CsCompile.LangVersion)" Optimize="%(CsCompile.Optimize)" OutputAssembly="$(IntDir)$(TargetName).netmodule" PdbFile="$(IntDir)$(TargetName).netmodule.pdb" TargetType="Module" TreatWarningsAsErrors="%(CsCompile.TreatWarningsAsErrors)" Utf8Output="%(CsCompile.Utf8Output)" WarningLevel="%(CsCompile.WarningLevel)" WarningsAsErrors="%(CsCompile.WarningsAsErrors)" WarningsNotAsErrors="%(CsCompile.WarningsNotAsErrors)" ToolExe="$(CscToolExe)" ToolPath="$(CscToolPath)" />
```

Comment:
I think I should stop caring how this `Csc` element is aligned. crosire cared about it and I kept it, but I'm starting to feel like caring how that `Csc` element is aligned is not worth it too much unless there's a way to keep Visual Studio from doing that.